File: app/src/main/java/com/sagar/watchnext/network/models/tv/Shows.java

package com.sagar.watchnext.network.models.tv;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Shows {

    @SerializedName("page")
    private int page;

    @SerializedName("results")
    private List<Show> shows;

    @SerializedName("total_results")
    private int totalResults;

    @SerializedName("total_pages")
    private int totalPages;

}

File: app/src/main/java/com/sagar/watchnext/network/models/tv/Shows.kt
package com.sagar.watchnext.network.models.tv

import com.google.gson.annotations.SerializedName

class Shows {

    @SerializedName("page")
    val page: Int = 0

    @SerializedName("results")
    val shows: List<Show>? = null

    @SerializedName("total_results")
    val totalResults: Int = 0

    @SerializedName("total_pages")
    val totalPages: Int = 0
}
